lib/livereload.js

```diff
@@ -97,6 +97,12 @@ function StartServer() {
                         return monkeyPatch() + snippet + match;
                     }
                 }
+            },
+            minify: false,
```

Comment:
I understand why you would want to specify defaults. How does this fix the bug that you mentioned if your value is same as the default?